device/celestica/x86_64-cel_seastone-r0/sonic_platform/module.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
```

Comment:
In the new platform API terminology, a "module" is a removable (i.e., modular) component, such as a line card or supervisor card, in a modular chassis. Thus, I believe this is an incorrect use of this class.

If there is no better class for this functionality, you could possibly create your own class which subclasses DeviceBase.